### PR TITLE
Issue #1395 - Dapper.Contrib inconsistent id field quoting with PostgreSQL

### PR DESCRIPTION
**Pull request by ryan-todd**

### CODE REVIEW

File: Dapper.Contrib/SqlMapperExtensions.cs [hunk 1]
```diff
@@ -1001,7 +1007,7 @@ public int Insert(IDbConnection connection, IDbTransaction transaction, int? com
                 if (!first)
                     sb.Append(", ");
                 first = false;
-                sb.Append(property.Name);
+                sb.Append($"\"{property.Name}\"");
```
Comment (icambron) [inline Dapper.Contrib/SqlMapperExtensions.cs]: isn't this the same as `adapter.AppendColumnName(sb, property.Name)`?

Comment (ryan-todd) [reply Dapper.Contrib/SqlMapperExtensions.cs]: Yep, not sure how I missed this one!

**Comment by rclarke2050** [comment]
can someone tell me when this is going in please?
I am reaching this issue now with lambda .NetCore on AWS Aurora.  Aurora names the fields in lowercase by default, even if i create the tables with Casing.

In the .net classes i use, they're in PascalCase, and for .Net SQL connections it doesnt matter. but for NpgsqlConnection, it totally doesnt like it.

Please can we get this PR finished?

**Comment by ryan-todd** [comment]
Not sure on the process as this is my first real contribution to a public repo - what's the next step to get this going?

**Comment by mgravell** [comment]
Hi; sorry, we (the green-lighters) are swamped; sorry if we didn't get to this yet. But; let's see what we can do here!

IMO the thing that is missing here is a test; something that demonstrates the problem being tackled and fixed, so we can have confidence in what it is doing. Is it possible to add a test for this? there is a `PostgresqlTests` class (or you can create a new class that extends `: TestBase<PostgresProvider>`)

**Comment by ryan-todd** [comment]
Hi mgravell, that's ok! I'll have a look at tests when I next get a chance, or if anyone else thinks they can add this, feel free.

**Comment by NickCraver** [comment]
I can't speak to Contrib's future plans here and if this is a good fit, but please see #1658 - Dapper.Contrib is not located at https://github.com/DapperLib/Dapper.Contrib (separate from this repository). 

Note: I based the Dapper.Contrib fork on the original codebase which hopefully makes porting pull requests against the main branch there much easier - the intent was to split, not throw away work the community wants to go with.

**Pull request closed without merging**